language/tools/disassembler/src/disassembler.rs

```diff
@@ -1067,6 +1067,7 @@ impl<'a, Location: Clone + Eq> Disassembler<'a, Location> {
         let name_opt = self.source_mapper.source_map.module_name_opt.as_ref();
         let name =
             name_opt.map(|(addr, n)| format!("{}.{}", addr.short_str_lossless(), n.to_string()));
+        let version = format!("Move bytecode v{}", self.source_mapper.bytecode.version());
```

Comment:
We are hoping for the disassembler to eventually round trip with the IR assembler/compiler. 

This is fine for now but flagging for @modocache 
Maybe we can do something like just putting it in a comment? (i.e. `// Move bytecode v<num>`)